### PR TITLE
Implement profile sapling key table

### PR DESCRIPTION
**Pull request by isabeltomb**
This PR implements the key table in the profile sapling

The functioning key table displays the keys associated with the authenticated user. The `New Key` button uses the original profile sapling form to generate and add new keys.

Paging is not yet implemented for the table so all new keys will be added to the bottom of the table.

The delete button in the key actions menu is also not implemented.

Copyright headers are added to the new files that did not have them previously.

**Testing:**

1. Run the splinter-ui using either the biome docker-compose or set the necessary environment variables and use the oauth docker-compose
`docker-compose -f docker-compose-biome.yaml up --build`
OR
`docker-compose -f docker-compose-oauth.yaml up --build`

2. In a browser navigate to `http://localhost:3030`

3. Register/log in and select the profile tab in the left nav bar

4. Click the `+ New Key` button and add a new key

5. View the new key in the key table on the profile page

6. Select the activate button (check mark) from the available key actions and activate the key
The key should now show `Active` in the status column and the activate option should no longer appear in the key actions

8. Add a second key using the `+ New Key` button
There should now be two keys in the key table

9. Activate the second key
The second key should now show `Active` in the status column and the first key's status should be empty indicating it is no longer active. The first key should now have the activate option in the key actions again.

### CODE REVIEW

File: saplings/profile/src/App.js [hunk 1]
```diff
@@ -18,14 +18,21 @@ import './App.scss';
...
       <Router>
         <Switch>
-          <Route exact path="/profile" component={Profile}/>
-          <Route exact path="/profile/new-key" component={NewKey}/>
+          <Route exact path="/profile">
+            <SaplingHeader name="Profile"/>
+            <Profile />
+          </Route>
+          <Route exact path="/profile/new-key">
+            <SaplingHeader name="Profile"/>
```
Comment (ltseeley) [inline saplings/profile/src/App.js]: Should this `name` attribute be something different?

**Review by shannynalayna — COMMENTED** [review]
I keep getting a console warning, `Form submission canceled because the form is not connected` when adding keys even when the form is submitted successfully

File: saplings/profile/src/App.js (hunk 1 above)
Comment (isabeltomb) [reply saplings/profile/src/App.js]: changed this to `saplingName` instead

**Comment by isabeltomb** [comment]
@shannynalayna I believe the console warning is coming from the submission button in `MultiStepForm` and will be resolved in the PR for implementing the new key form

**Review by shannynalayna — APPROVED** [review]

**Review by ltseeley — APPROVED** [review]

**Pull request merged**